h/static/styles/thread.scss

```diff
@@ -20,7 +20,7 @@ $thread-padding: $annotation-card-left-padding;
   align-items: center;
 
   &__label {
-    max-width: 160px;
+    max-width: 180px;
```

Comment:
You can't tell from looking at this what the intended visual effect is, that a line break should appear after the end of the sentence. Any reason not to just use `<br>` tags in the template?